history.rst

```diff
@@ -5,6 +5,37 @@
 OMERO version history
 =====================
 
+
+5.6.14 (November 2024)
+----------------------
+
+This release includes the following upgrade of the OMERO.server Java components:
+
+- omero-gateway-java 5.10.0
+- omero-blitz 5.8.0
+- omero-server 5.7.0
+- omero-renderer 5.6.0
+- omero-romio 5.7.0
+- omero-common 5.7.0
+- omero-model 5.7.0
+
+
+Improvements include:
+
+- Bump slf4j dependencies. See https://github.com/ome/openmicroscopy/pull/6409
+- Add new LUTs See https://github.com/ome/openmicroscopy/pull/6398 and https://github.com/ome/omero-web/pull/581
```

Comment:
Maybe use the non acronym `lookup tables`? Should the OMERO.web PR be linked here as this history is primarily about the server ?

Alternatively, if this announcement is meant to cover server & web, this could be a bit more verbose about the improved lookup table functionalities (new server side tables, new dynamic endpoint).